openmetadata-spec/src/main/resources/json/schema/entity/services/connections/testConnectionResult.json

```diff
@@ -28,6 +28,11 @@
           "description": "Results or exceptions to be shared after running the test.",
           "type": "string",
           "default": null
+        },
+        "errorLog": {
```

Comment:
this lgtm. Could we update the `message` description as "this comes from our test definition"? to not have confusion with user-friendly message vs log